### PR TITLE
Add sheets add-on documentation.

### PR DESCRIPTION
**Pull request by mvashishtha**
Document the place DCID search, as well as all of the custom functions.

### CODE REVIEW

File: api/sheets/find_dcid.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Find DCID
+nav_order: 4
```
Comment (shifucun) [inline api/sheets/find_dcid.md:4]: need to update the nav_order for all the files. They are the order appear on the ui

File: api/sheets/find_dcid.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+parent: REST
```
Comment (shifucun) [inline api/sheets/find_dcid.md:5]: parent should be Sheet?

File: api/sheets/get_name.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+layout: default
+title: Get the name of a Data Commons node given its dcid.
+nav_order: 4
+parent: SHEETS
+grand_parent: API
+---
+
+# Get the name of a node.
+
+## =c(dcids)
```
Comment (shifucun) [inline api/sheets/get_name.md:11]: what is =c ?

File: api/sheets/get_dcid.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: Get place DCID
+nav_order: 4
+parent: REST
+grand_parent: API
+---
+
```
Comment (shifucun) [inline api/sheets/get_dcid.md:8]: This seems very similar to find_dcid. May point out the difference?

File: api/sheets/get_name.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Given a cell or a column range of cells with DCIDs, return the names of those dcids.
+
+**Arguments**
+*    `dcids (str)` - DCIDs to get the names of
```
Comment (shifucun) [inline api/sheets/get_name.md:16]: this is a python style annotation. In sheets I guess there is no strict way of typing?

File: api/sheets/get_name.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+in cells B2 and B3 respectively.
+
+### Input
+
+![](/assets/sheets_get_name_input.png)
+
+### Output
+
+fdsfs
```
Comment (shifucun) [inline api/sheets/get_name.md:44]: remove

File: api/sheets/get_variable.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+nav_order: 4
+parent: SHEETS
+grand_parent: API
+---
+
+# Get the value of a statistical variable at a given place and time.
+
+## =DCGET(dcids, variable, date)
+
+Given a list of [Place](https://browser.datacommons.org/kg?dcid=Place) DCIDs, a [StatisticalVariable](https://browser.datacommons.org/kg?dcid=StatisticalVariable), and a date, get the value of the variable at those places on that date.
```
Comment (shifucun) [inline api/sheets/get_variable.md:13]: statistics values

File: api/sheets/places_in.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+## =DCPLACESIN(dcids, placeType)
+
+Given a list of [`Place`](https://browser.datacommons.org/kg?dcid=Place) DCIDs, return the DCIDs of places of a specified type
+contained within those places.
+
+## Examples
+
+**Arguments**
+*    `dcids (str)` - place DCIDs to get places within
```
Comment (shifucun) [inline api/sheets/places_in.md:19]: extra space here

File: api/sheets/find_dcid.md (hunk 2 above)
Comment (mvashishtha) [reply api/sheets/find_dcid.md:5]: Done.

File: api/sheets/find_dcid.md (hunk 1 above)
Comment (mvashishtha) [reply api/sheets/find_dcid.md]: Done.

File: api/sheets/find_dcid.md (hunk 2 above)
Comment (mvashishtha) [reply api/sheets/find_dcid.md]: Done.

File: api/sheets/get_name.md (hunk 3 above)
Comment (mvashishtha) [reply api/sheets/get_name.md]: That should be DCGETNAME, I was trying to copy/paste :)

File: api/sheets/get_dcid.md (hunk 4 above)
Comment (mvashishtha) [reply api/sheets/get_dcid.md:8]: Sorry, I renamed find_dcid to get_dcid but I didn't delete the original in the commit. I deleted it now.

File: api/sheets/get_name.md (hunk 5 above)
Comment (mvashishtha) [reply api/sheets/get_name.md]: Yeah, I got rid of all the type annotations.

File: api/sheets/get_name.md (hunk 6 above)
Comment (mvashishtha) [reply api/sheets/get_name.md]: Done.

File: api/sheets/get_variable.md (hunk 7 above)
Comment (mvashishtha) [reply api/sheets/get_variable.md:13]: What do you mean by this?

File: api/sheets/places_in.md (hunk 8 above)
Comment (mvashishtha) [reply api/sheets/places_in.md]: Deleted.

File: api/sheets/get_variable.md (hunk 7 above)
Comment (shifucun) [reply api/sheets/get_variable.md:13]: "get the value of the variable" is a bit mis-leading here? We want to say the stats(data) corresponding to the stats_var

Comment (mvashishtha) [reply api/sheets/get_variable.md]: I changed it to "get the measurements of that variable at those places on that date." Sorry for the force push, I had to do it to fix the CLA problem that googlebot complained about.

**Pull request merged**